app/assets/stylesheets/modules/sitter.scss

```diff
@@ -129,13 +154,26 @@
         font-weight: $sitter-headline-weight;
       }
 
+      #image-upload {
```

Comment:
Avoid using id selectors<br>Selector should have depth of applicability no greater than 2, but was 4